reconcile/terraform_resources.py

```diff
@@ -394,32 +418,57 @@ def fetch_current_state(dry_run, namespaces, thread_pool_size,
 
 
 def init_working_dirs(accounts, thread_pool_size,
-                      oc_map=None, settings=None):
+                      aws=None,
+                      default_ci_account=None,
+                      settings=None):
     ts = Terrascript(QONTRACT_INTEGRATION,
                      QONTRACT_TF_PREFIX,
                      thread_pool_size,
                      accounts,
+                     aws=aws,
+                     default_ci_account=default_ci_account,
                      settings=settings)
     working_dirs = ts.dump()
     return ts, working_dirs
 
 
+def get_default_ci_account(accounts):
+    ci_accounts = [a for a in accounts if a.get('defaultCI')]
+    if len(ci_accounts) == 0:
+        raise KeyError('default ci accounts not found')
+    if len(ci_accounts) > 1:
+        account_names = [a['name'] for a in ci_accounts]
+        raise KeyError(
+            'multiple default ci accounts found: '
+            f'{account_names}'
+        )
+    return ci_accounts[0]
```

Comment:
this can probably move to queries? maybe around here: https://github.com/app-sre/qontract-reconcile/blob/8a673b521dfedea89fc6a3be1cebccddd6d01879/reconcile/queries.py#L329-L338